source/_integrations/ubus.markdown

```diff
@@ -17,18 +17,18 @@ Before this scanner can be used you have to install the ubus RPC package on Open
 opkg install rpcd-mod-file
 ```
 
-For OpenWrt version 18.06.x the package uhttpd-mod-ubus should also be installed:
+For OpenWrt version 18.06 and higher, the package uhttpd-mod-ubus should also be installed:
 
 ```bash
 opkg install uhttpd-mod-ubus
 ```
 
-And create a read-only user to be used by setting up the ACL file `/usr/share/rpcd/acl.d/user.json`.
+And create a read-only user to be used by setting up the ACL file `/usr/share/rpcd/acl.d/hassio.json`.
```

Comment:
```suggestion
And create a read-only user to be used by setting up the ACL file `/usr/share/rpcd/acl.d/hassio.json`.
```

Please remove/avoid the use of the term `hassio` as we are phasing out that term, use `homeassistant` instead.